src/components/ReviewInfoText/index.tsx

```diff
@@ -42,17 +48,39 @@ export const ReviewInfoText = ({
 
   const transactionsToGo = safeNonceNumber - nonce
 
+  useEffect(() => {
+    const fetchRecommendedNonce = async () => {
+      try {
+        const recommendedNonce = (await getRecommendedNonce(safeAddress)).toString()
+        setRecommendedNonce(recommendedNonce)
+      } catch (e) {
+        return
+      }
+    }
+    fetchRecommendedNonce()
+  }, [safeAddress])
```

Comment:
Needs `setRecommendedNonce`. Strange that lint didn't pick this up.

Reply:
I cleaned the dependency array and imported the dependencies automatically. I remember wondering if the `useState` setters belong in the dependency array once they don't really change.
Should we add it anyway?